config/routes.rb

```diff
@@ -296,6 +296,9 @@
         resources :reject_applications, controller: "reject_applications", only: %i[index create show] do
           post "run", on: :member
         end
+        resources :update_and_verify_trns, controller: "update_and_verify_trns", only: %i[index create show] do
```

Comment:
minor nit - can you add a blank line before this since these are multiple code blocks